src/HttpServer.ts

```diff
@@ -69,7 +69,7 @@ export class HttpServer {
     this._app.use(bodyParser.json());
     /* eslint-disable @typescript-eslint/no-misused-promises */
     this._app.post('/', this.rootHandler.bind(this));
-    this._app.get('/getaddr', this.pingHandler.bind(this));
+    this._app.get('/get-chain-info', this.getChainInfo.bind(this));
```

Comment:
```suggestion
    this._app.get('/chain-info', this.getChainInfo.bind(this));
```

Reply:
btw if you define getChainInfo as arrow function you won't need to bind it ;)

Reply:
cool, let me try it :)